### PR TITLE
[modular] Dogginos! A service ERT that delivers Pizza.

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

The title.

## Why It's Good For The Game

Flavour is always good. 
This is the first in a small project of mine of making specific ERTs for various things, this one is a service/joke ERT. 
If an admin is feeling particularly evil, they can spawn a Pizza ert, who's means of defence are...Pizza and kitchen knives, the Manager gets a 1000 degree knife that ignites someone on hit.

image
image
image
coats are virgo stuff
image

## Changelog
:cl:
add: Dogginos has established a local branch in the Skyrat sector.
/:cl:

### CODE REVIEW

**Comment by thestubborn** [comment]
semi wip, kinda

**Comment by FlamingLily** [comment]
unironic new antag idea, "Pizza Cat", the blood rival to Dogginoes.

They come decked out in BRH, and only wield pizza bombs.

**Comment by thestubborn** [comment]
> 
> 
> unironic new antag idea, "Pizza Cat", the blood rival to Dogginoes.
> 
> They come decked out in BRH, and only wield pizza bombs.

i was trying to think of how to fit schrodinger's, the name I came up for the cat pizza brand in, honestly

**Comment by Gandalf2k15** [comment]
based

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
WW2 Germany stuff involving Nat Seas is TOS I think?
Best be safe.

File: modular_skyrat/modules/additionalert/pizza/code/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/obj/item/clothing/head/pizza
+	name = "dogginos manager hat"
+	desc = "Seriously, why does a pizza place wear nazi chic?"
```
[the remark this answers is not in the document]
Comment (thestubborn) [reply modular_skyrat/modules/additionalert/pizza/code/modules/clothing/head/head.dm]: done

**Pull request merged**